pkg/policy/l4.go

```diff
@@ -43,8 +48,49 @@ type L4Filter struct {
 	Ingress bool
 }
 
-// CreateL4Filter creates an L4Filter based on an api.PortRule and api.PortProtocol
-func CreateL4Filter(rule api.PortRule, port api.PortProtocol, direction string, protocol string) L4Filter {
+// createHTTPRule creates an AuxRule based on an api.PortRuleHTTP.
```

Comment:
Can you write an example of what the result would be for a particular `api.PortRuleHTTP`?